omero/sysadmins/unix/walkthrough/setup_nginx_centos6_selinux.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [ $(getenforce) != Disabled ]; then
```

Comment:
```
bash-4.1# rpm -qf /usr/sbin/getenforce
libselinux-utils-2.0.94-5.8.el6.x86_64
bash-4.1# yum remove libselinux-utils
...
================================================================================
 Package             Arch   Version                  Repository            Size
================================================================================
Removing:
 libselinux-utils    x86_64 2.0.94-5.8.el6           @CentOS/$releasever   61 k
Removing for dependencies:
 ice-servers         x86_64 3.5.1-1.el6              @zeroc-ice            12 M
 initscripts         x86_64 9.03.46-1.el6.centos.1   @Updates/$releasever 5.5 M
 iproute             x86_64 2.6.32-33.el6_6          @Updates/$releasever 928 k
 iptables            x86_64 1.4.7-14.el6             @CentOS/$releasever  836 k
 iputils             x86_64 20071127-17.el6_4.2      @CentOS/$releasever  292 k
 nginx               x86_64 1.6.2-1.el6.ngx          @nginx               828 k
 policycoreutils     x86_64 2.0.83-19.47.el6_6.1     @Updates/$releasever 3.5 M
 postgresql93        x86_64 9.3.5-1PGDG.rhel6        @pgdg93              5.2 M
 postgresql93-server x86_64 9.3.5-1PGDG.rhel6        @pgdg93               16 M
 udev                x86_64 147-2.57.el6             @CentOS/$releasever  1.2 M
 util-linux-ng       x86_64 2.17.2-12.18.el6         @CentOS/$releasever  5.8 M

Transaction Summary
================================================================================
Remove       12 Package(s)
```
So a fairly low-level dependency.

https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/6/html/Security-Enhanced_Linux/sect-Security-Enhanced_Linux-Working_with_SELinux-Enabling_and_Disabling_SELinux.html
`getenforce` should return one of

- `Enforcing`: SELinux is active (default in CentOS^)- changes in the default policy in CentOS6.6 cause problems with NGinx as reported in the forums
- `Permissive`: warns about operations which would be blocked in enforcing mode (should really be for testing only)
- `Disabled`